config/examples/backend_redis.yml

```diff
@@ -1,9 +1,10 @@
 base: &default
   url: "<%= ENV.fetch('BACKEND_REDIS_URL', 'redis://localhost:6379/6') %>"
   timeout: <%= ENV.fetch('BACKEND_REDIS_TIMEOUT', 1) %>
-  pool_size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
+  size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
```

Comment:
Just a reminder, did you create a PR for SaaS with this?

Reply:
No, but I think we don't need it just now, because it's backwards compatible: https://github.com/3scale/porta/pull/3576/files#diff-ad73aa14ad3c1b2d53ac0808de8bbf07f16747d251bf28258622caa5bc7b21c5R9

So, we can update it when it is already deployed.